src/core/core.ts

```diff
@@ -21,23 +21,25 @@ export class Core implements OnInit {
   @ViewChild(Nav) nav: Nav;
 
 
-  constructor(private appCtrl: App) {
-
-  }
+  // constructor(private appCtrl: App) { }
 
   ngOnInit() {
-    /** TODO: Auto login */
-
-    /** Subscribe to page changes event */
-    this.appCtrl.viewWillEnter
-      .filter((page: ViewController) => !page.enableBack())
-      .subscribe((page: ViewController) => {
-
-        /** Check if user is authenticated */
-        if (!this.state.authenticated && page.name !== 'LoginPage') {
-          this.nav.setRoot('LoginPage');
-        }
-      });
+    /** Disable auth guard for development purpose */
```

Comment:
I disabled the auth guard in this PR, to prevent redirection while developing other tasks.